syntax = "proto3";
package ni.measurementlink.examples.v1;

message Configurations {
    string pin_name = 1;
    int32 measurement_type = 2; // really an enum
    double range = 3;
    double resolution_digits = 4;
}

message Outputs {
    double measured_value = 1;
    bool signal_out_of_range = 2;
    double absolute_resolution = 3;
}